features/multi_candidate_consumer.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Support multi-candidate consumer
+
+  Scenario: A consumer can publish to a single candidate
+    Given the Consumer owned by groovy with Consumer Token e0bf422d63d65ef1f4fe573a0d461d695edef45a541f07f3747ad37188329368 for candidate groovy
+    And the header Consumer-Key 5f202e7ab75f00af194c61cc07ae6b0c is presented
+    And the header Consumer-Token e0bf422d63d65ef1f4fe573a0d461d695edef45a541f07f3747ad37188329368 is presented
+    And the remote release service returns a CREATED response:
+    """
+          |{
+          |  "status": 201,
+          |  "id": "5426b98dba78e60054fe482f",
+          |  "message": "released groovy version: 2.3.6"
+          |}
+    """
+    When posting JSON on the /release endpoint:
+    """
+          |{
+          |  "candidate" : "groovy",
+          |  "version" : "2.3.6",
+          |  "url" : "http://hostname/groovy-binary-2.3.6.zip"
+          |}
+    """
+    Then the status received is CREATED
+    And the remote release service expects a Consumer header groovy
+
+  Scenario: A consumer can publish to multiple candidates
+    Given the Consumer owned by groovy with Consumer Token e0bf422d63d65ef1f4fe573a0d461d695edef45a541f07f3747ad37188329368 for candidate groovy,grails
+    And the header Consumer-Key 5f202e7ab75f00af194c61cc07ae6b0c is presented
+    And the header Consumer-Token e0bf422d63d65ef1f4fe573a0d461d695edef45a541f07f3747ad37188329368 is presented
+    And the remote release service returns a CREATED response:
```

Comment:
Same question as above: Without diving into the implementation its hard to understand why the release service returns anything.

Reply:
Same as above. I'll reword that step to mentioning the fact that it is a proxied response.